cli/Valet/PhpFpm.php

```diff
@@ -192,8 +253,31 @@ public function useVersion($version, $force = false)
             $this->brew->ensureInstalled($version, [], $this->taps);
         }
 
-        // Unlink the current php if there is one
+        // Delete old Valet sock files, install the new version, and, if this is a global change, unlink and link PHP
+        if ($directory && $site) {
+            $customPhpVersion = $this->site->customPhpVersion($site); // Example output: "74"
```

Comment:
Is this referencing the *preexisting* custom PHP version, if set?

If so, maybe this variable should be named`$oldCustomPhpVersion`?